src/style.css

```
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/avenir-book.ttf");
+  font-style: normal;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf");
+  font-style: italic;
+  font-weight: normal;
+}
+
+body {
+  box-sizing: border-box;
+}
```

It'd be better to make it with a `*` selector to make it universal for all elements
```suggestion
* {
  box-sizing: border-box;
}
```